module/profiler/profiler.go

```diff
@@ -116,11 +138,11 @@ func newProfileFunc(name string) profileFunc {
 	}
 }
 
-func (p *AutoProfiler) pprofHeap(w io.Writer) error {
+func (p *AutoProfiler) pprofAllocs(w io.Writer) error {
```

Comment:
if allocs and heap profile different things, we should profile both.

Reply:
will mention this in the description: alloc and heap are the same. the only difference is which sample is marked as "default".